dask_glm/normal.py

```diff
@@ -0,0 +1,36 @@
+from __future__ import absolute_import, division, print_function
+
+from dask_glm.utils import dot
+
+
+try:
+    from numba import jit
+except ImportError:
+    def jit(*args, **kwargs):
+        def _(func):
+            return func
+        return _
+
+
+def loglike(Xbeta, y):
+    return ((y - Xbeta)**2).sum()
+
+
+def pointwise_loss(beta, X, y):
+    beta, y = beta.ravel(), y.ravel()
+    Xbeta = X.dot(beta)
+    return loglike(Xbeta, y)
+
+
+def pointwise_gradient(beta, X, y):
+    beta, y = beta.ravel(), y.ravel()
+    Xbeta = X.dot(beta)
+    return gradient(Xbeta, X, y)
+
+
+def gradient(Xbeta, X, y):
+    return 2 *  dot(X.T, Xbeta) - 2 * dot(X.T, y)
+
+
+def hessian(Xbeta, X):
+    return dot(X.T, X)
```

Comment:
After thinking over @mcg1969's [comment on the API](https://github.com/dask/dask-glm/issues/11#issuecomment-281191366), I'm on board with having a class with static methods that hold these functions (loglike, gradient, hessian, etc.), one class for each unregularized GLM family.

Separately, we can have a "model" class (also probably one for each family) that has the fit / summary / initialization routines.